Resources/Prototypes/Entities/Structures/Wallmounts/intercom.yml

```diff
@@ -220,7 +220,15 @@
   id: IntercomCommand
   parent: Intercom
   suffix: Command
+  description: An intercom. It's been reinforced with metal from security helmets, making it a bitch-and-a-half to open.
```

Comment:
make a new description since this one is obviously tailored to the security intercom.